lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+
+class Player::Human < Player
+
+  def move(board)
+    puts "Where would you like to go (choose a number from 1-9)?"
+    input = gets.chomp
+  end
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(token, board)
  puts "Where would you like to go (choose a number from 1-9)?"
  gets.chomp
end
```